modules/5_install/install.tf

```diff
@@ -63,6 +63,9 @@ locals {
         chrony_config_servers    = var.chrony_config_servers
         chrony_allow_range       = var.cidr
         cni_network_provider     = var.cni_network_provider
+        cluster_network_cidr     = var.cluster_network_cidr
+        cluster_network_hostprefix = var.cluster_network_hostprefix
```

Comment:
can you please format this section to look uniform, will just need to add additional spaces to rest of vars.

Reply:
just this minor formatting change, rest lgtm.